WBooks/Book Info/BookInfoView.xib

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If the view does not have anything, I think you don't need the .xib nor the view's class, UIViewController has it's own view.